content/documentation/edge/features/index.adoc

```diff
@@ -143,55 +188,117 @@ Kiali provides filtered list views of all your service mesh definitions. Each vi
 
-The service detail view shows request and response metrics for inbound traffic.
 ++++
-<a class="video-popup" href="/images/documentation/features/detailed_metrics.mp4" title="Metric Charts">
-    <video autoplay muted loop width="600px" src="/images/documentation/features/detailed_metrics_thumb.mp4" style="display:block;margin:0 auto;" />
-</a>
+<div style="display: flex;">
+ <span style="margin: 0 auto;">
+  <a class="image-popup-fit-height" href="/images/documentation/features/overview-app-v1.22.0.png" title="App Overview">
+    <img src="/images/documentation/features/overview-app-thumb-v1.22.0.png" style="width: 660px; display:inline;margin: 0 auto;" />
+  </a>
+  <a class="image-popup-fit-height" href="/images/documentation/features/overview-workload-v1.22.0.png" title="Workload Overview">
+    <img src="/images/documentation/features/overview-workload-thumb-v1.22.0.png" style="width: 660px; display:inline;margin: 0 auto;" />
+  </a>
+</span>
+</div>
+<div style="display: flex;">
+ <span style="margin: 0 auto;">
+  <a class="image-popup-fit-height" href="/images/documentation/features/overview-service-v1.22.0.png" title="Service Overview">
+    <img src="/images/documentation/features/overview-service-thumb-v1.22.0.png" style="width: 660px; display:inline;margin: 0 auto;" />
+  </a>
+ </span>
+</div>
 ++++
 
 {empty} +
 
-==== Detail: Services
-The service detail view shows the user the workloads running the service. It also shows the Istio traffic routing configuration, VirtualServices and DestinationRules, associated with the service.
+==== Detail: Traffic
+
+The traffic tab collects all connections from and to a specific Application, Workload and/or Service.
+
+It groups them in Inbound and Outbound tables prividing traffic details and links to specific detailed metrics.
 
-Kiali provides access to YAML definitions and allows modification and deletion access for authorized users. It provides wizards to assist in common configurations and performs additional validation on VirtualServices to detect misconfigured routes.
 ++++
-<a class="video-popup" href="/images/documentation/features/services_traffic_routing.mp4" title="Service Details">
-    <video autoplay muted loop width="600px" src="/images/documentation/features/services_traffic_routing_thumb.mp4" style="display:block;margin:0 auto;" />
-</a>
+ <a class="image-popup-fit-height" href="/images/documentation/features/traffic-v1.22.0.png" title="Traffic tab">
+  <img src="/images/documentation/features/traffic-thumb-v1.22.0.png" style="display:block;margin: 0 auto;" />
+ </a>
 ++++
 
 {empty} +
 
-==== Detail: Workloads
-Kiali performs several validations on workload configuration:
+==== Detail: Logs
 
-* Are Istio sidecars deployed?
+The Workload detail offer a special Logs tab. Kiali allows to examine at same time application containers and sidecar logs of a selected Pod.
```

Comment:
(optional/suggested)  for second sentence:

Kiali allows side-by-side viewing of both the application and the proxy logs of a selected pod.